app/assets/json/translations/react/tl.json

```diff
@@ -915,6 +915,7 @@
   "listings.stats.salesPrice": "Presyo sa pagbebenta",
   "listings.stats.tableLabel": "Talahanayan ng pagpepresyo para sa %{numInHouseholdString}, %{amiPercentString}",
   "listings.stats.upToPercentAmi": "Hanggang %{amiPercent}% AMI",
+  "listings.stats.upToPercentAmi.fullText": "Hanggang %{amiPercent}% AMI (Area Median Income)",
```

Comment:
Yea exactly. I noticed the current behavior is that if a key is not found in one of the other language files then it falls back to the english file. I debated adding them to all files regardless if they are the same as english or not that way they would explicitly be in the file rather than having to know the fall back strategy. But I decided to leave it the way it currently is.